ramalama/huggingface.py

```diff
@@ -69,6 +67,24 @@ def pull(self, args):
         symlink_dir = os.path.dirname(model_path)
         os.makedirs(symlink_dir, exist_ok=True)
 
+        try:
+            return self.url_pull(args, model_path, directory_path)
+        except (urllib.error.HTTPError, urllib.error.URLError, KeyError) as e:
+            if not self.hf_cli_available:
```

Comment:
This isn't working for me, I'm ending up with it showing my hf_cli is not available even though it is.

Reply:
```suggestion
            if self.hf_cli_available:
```
not is reversing the logic, right?

Reply:
yep